docs/modules/trino/pages/usage_guide/operations/graceful-shutdown.adoc

```diff
@@ -44,7 +44,7 @@ Until then, you have to use configOverrides to enable it.
 == Kubernetes cluster requirements
 Pods need to have the ability to take as long as they need to gracefully shut down without getting killed.
 
-Imagine the situation that you set the graceful shutdown period to 24 hours (using `spec.clusterConfig.gracefulShutdownSeconds: 86400`).
+Imagine the situation that you set the graceful shutdown period to 24 hours (using `spec.clusterConfig.gracefulShutdownTimeout: 86400s`).
```

Comment:
Wouldn't it make more sense to say 1d or 24h in this example?

Reply:
probably yes :) https://github.com/stackabletech/trino-operator/pull/466/commits/967d077cd35303c969d7a2e7a780acb25783e849